csharp/compatibility_tests/v3.0.0/test.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+function run_test() {
+  # Generate test proto files.
+  ./protoc_1 -Iprotos/src -I../../../src/ --csharp_out=src/Google.Protobuf.Test \
+    --csharp_opt=base_namespace=Google.Protobuf \
+    protos/src/google/protobuf/map_unittest_proto3.proto \
+    protos/src/google/protobuf/unittest_import_proto3.proto \
+    protos/src/google/protobuf/unittest_import_public_proto3.proto \
+    protos/src/google/protobuf/unittest_well_known_types.proto
+
+  ./protoc_1 -Iprotos/csharp --csharp_out=src/Google.Protobuf.Test \
+    --csharp_opt=base_namespace=UnitTest.Issues \
+    protos/csharp/protos/unittest_issues.proto
+
+  ./protoc_2 -Iprotos/src --csharp_out=src/Google.Protobuf.Test \
```

Comment:
It's not clear to me why just this one uses protoc_2, when the others use protoc_1. Is is because this is the only file that's changing? (I've read the comments on the tests themselves, and I'm none the wiser I'm afraid!)